### ISSUE TITLE
replace pyfits with its implementation in astropy (Trac #782)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#776]
integrate it nicely in cmake a la numpy

affects clsim and photospline

should hopefuly make doc errors STFU

Migrated from https://code.icecube.wisc.edu/ticket/782
```json
{
    "status": "closed", 
    "changetime": "2015-06-24T11:27:50", 
    "description": "integrate it nicely in cmake a la numpy\n\naffects clsim and photospline\n\nshould hopefuly make doc errors STFU", 
    "reporter": "nega", 
    "cc": "jvansanten", 
    "resolution": "fixed", 
    "_ts": "1435145270717798", 
    "component": "combo simulation", 
    "summary": "replace pyfits with its implementation in astropy", 
    "priority": "minor", 
    "keywords": "clsim photospline documentation", 
    "time": "2014-10-13T21:02:49", 
    "milestone": "", 
    "owner": "nega", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150420T01:33:58: 
* nega changed cc from "jvansanten, claudio.kopper" to ""
* nega changed priority from "normal" to "minor"

**Comment by icecube-trac** [comment]
Trac update at 20150624T02:21:31: nega changed cc from "" to "jvansanten"

**Comment by icecube-trac** [comment]
Trac update at 20150624T11:27:50: 
* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"
* jvansanten commented:

> In changeset 134370:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="134370"
> PyFITS is deprecated. Fall back to astropy.io.fits if it can't be found. Closes #776.
> ```
